From 046d03c87ecce5db074eae6ffa2d5298c5f7d5a7 Mon Sep 17 00:00:00 2001
From: Leon Anavi <leon.anavi@konsulko.com>
Date: Tue, 12 Dec 2023 13:53:37 +0200
Subject: [PATCH] drivers/pinctrl/bcm/Kconfig: Fix BCM2712 help

Replace "Broadcom BCM2835 GPIO" with "Broadcom BCM2712 PINCONF"
in the help message. This work was sponsored by GOVCERT.LU.

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 drivers/pinctrl/bcm/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/pinctrl/bcm/Kconfig
+++ b/drivers/pinctrl/bcm/Kconfig
@@ -10,7 +10,7 @@ config PINCTRL_BCM2712
 	select PINCONF
 	select GENERIC_PINCONF
 	help
-	   Say Y here to enable the Broadcom BCM2835 GPIO driver.
+	   Say Y here to enable the Broadcom BCM2712 PINCONF driver.
 
 config PINCTRL_BCM281XX
 	bool "Broadcom BCM281xx pinctrl driver"
